layers/meta-balena-beaglebone/recipes-kernel/linux/linux-bb.org_git.bbappend

```diff
@@ -1,2 +1,7 @@
 inherit kernel-balena
 
+# BRANCH:aarch64 = "v5.10.168-ti-arm64-r111"
```

Comment:
These comments can be removed

Reply:
Looks like @floion has already removed this comment in the new PR (https://github.com/balena-os/balena-beaglebone/pull/682)

Reply:
Let's have them removed from here. PR 682 is just for testing

Reply:
Not resolved